.github/workflows/tests.yml

```diff
@@ -90,6 +90,13 @@ jobs:
             }
             core.setOutput('logs-dir', '/github/home/.cache/Homebrew/Logs')
 
+            if (label_names.includes('CI-linux-self-hosted')) {
+              core.setOutput('linux-runner', 'linux-self-hosted-1')
+            } else {
+              core.setOutput('linux-runner', 'ubuntu-22.04')
+              core.setOutput('logs-dir', '/github/home/.cache/Homebrew/Logs')
+            }
```

Comment:
This needs to be combined with the `if` block above. We should also do `core.setOutput('logs-dir', ...)` if and only if the runner is not self-hosted.

Reply:
👍